pkg/finality-grandpa/lib.go

```diff
@@ -80,18 +80,13 @@ func (m Message[H, N]) Target() HashNumber[H, N] {
 	}
 }
 
-// Value returns the message constrained by `Messages`
-func (m Message[H, N]) Value() any {
-	return m.value
-}
-
 // Messages is the interface constraint for `Message`
 type Messages[Hash, Number any] interface {
 	Prevote[Hash, Number] | Precommit[Hash, Number] | PrimaryPropose[Hash, Number]
 }
 
 func setMessage[Hash, Number any, T Messages[Hash, Number]](m *Message[Hash, Number], val T) {
```

Comment:
I think I left this code pass in the previous reviews but I think this function is useless since you can just set the value directly in the `newMessage` func

```go 

func newMessage[Hash, Number any, T Messages[Hash, Number]](val T) (m Message[Hash, Number]) {
	return Message[Hash, Number]{
            Value: val
        }
}

```

Reply:
I plan to talk to Tim about this section of code in our sync, so I will bring this up as well